hangman.rb

```diff
@@ -0,0 +1,124 @@
+class Hangman
+
+  def initialize
+    @now_playing = true
+    @count = 0
+
+    # Board Display
+    @row1 = "|     _________"
+    @row2 = "|     |/      |"
+    @row3 = "|     |      "
+    @row4 = "|     |      "
+    @row5 = "|     |       "
+    @row6 = "|     |      "
+    @row7 = "|     |"
+    @row8 = "| ____|_____"
+
+    @board = [@row1, @row2, @row3, @row4, @row5, @row6, @row7, @row8]
+
+    @dictionary = ["hotdog", "elephant", "computer", "developer", "baby", "dragon", "classmate", "program", "teamwork", "fun"]
+
+    start
+  end
+
+  def start
+    puts "Let's play a game of Hangman!"
+    puts @board
+
+    pick_word
+    @word = ["_ "] * @answer_letters.length
+    puts @word.join
+    play
+  end
+
+  def play
+    while @now_playing == true
+      get_guess
+      check_guess
+      puts @board
+      puts @word.join
+      win
+    end
+  end
+
+  def pick_word
+    #randomly selects word
+    answer = @dictionary[rand(0...@dictionary.length)]
+    #convert individual letters of word to array
```

Comment:
Be very careful of off by one errors. They are common in computer programming. I ran these things in IRB to confirm the problem:

2.2.2 :001 > @dictionary = ["hotdog", "elephant", "computer", "developer", "baby", "dragon", "classmate", "program", "teamwork", "fun"]
 => ["hotdog", "elephant", "computer", "developer", "baby", "dragon", "classmate", "program", "teamwork", "fun"] 
2.2.2 :002 > @dictionary.length
 => 10 
2.2.2 :003 > @dictionary[10]
 => nil 

I think you can take this one from here but feel free to tag me in a response if you want to ask questions about it.